workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/ActionNode.java

```diff
@@ -93,15 +93,7 @@ public ProcessInfo getProcessInfo() {
     public void setProcessInfo(ProcessInfo processInfo) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "setProcessInfo" is 27 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)